shellstub.py

```
@@ -0,0 +1,149 @@
+from __future__ import annotations
+
+"""Minimal framework for stubbing command line tools in tests."""
+
+import json
+import os
+import sys
+import textwrap
```

Ruff (0.11.9)

Module level import not at top of file

(E402)